### DISCUSSION TITLE
Move the Tree Style Tab sidebar to the left

### DISCUSSION THREAD

**Discussion by DemDemD** [piroor/treestyletab#3330] [Q&A]
Hi all,
    I've just installed the Tree Style Tab Sidebar and it gave me the option to place it to the right of FireFox.  However, the option to move it to Left or Right is no longer there.  Am I missing something?  I want to place the sidebar to the left of FireFox now.

Thanks in advance.

**Comment by irvinm** [comment]
![image](https://user-images.githubusercontent.com/979729/235470200-9ff315d8-a216-42e3-99ac-dcf31146a50c.png)

**Reply by DemDemD** [reply]
Thank you.  That's just to put the "X" on the right or left of the listed tabs.  I'm looking for a way to move the entire sidebar to the left or right.  Right now, it's stuck to the right of the browser and I want it to move to left.

I've figured it out by going into "about:config" to set the "sidebar.position_start" to "true"

**Reply by irvinm** [reply]
LOL, sorry about that ... :)  Glad you figured it out.

**Comment by DemDemD** [comment]
I've figured it out by going into "about:config" to set the "sidebar.position_start" to "true"
